### PR TITLE
docs: show parameters information in docs

### PR DESCRIPTION
**Pull request by Duologic**
Adds to #122, uses https://github.com/jsonnet-libs/docsonnet/pull/56

Also adds CSS to make things look a lot more concise. The CSS has comments that explain each change, see first commit.

grafonnet-docs-layout

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Jsonnet library for generating Grafana dashboards.
 
 ## Introduction
 
-This library depends on JSON Schemas generated by [Grok](https://github.com/grafana/grok),
+This library is generated from JSON Schemas generated by [Grok](https://github.com/grafana/grok),
```
Comment (Duologic) [inline README.md:7]: drive-by change

**Review by julienduchesne — APPROVED** [review]
Looks great!
